code/lib/cli-storybook/src/doctor/getIncompatibleStorybookPackages.ts

```diff
@@ -112,12 +126,24 @@ export const getIncompatiblePackagesSummary = (
       )} as they depend on different major versions of Storybook packages:`
```

Comment:
Can you maybe update the text to this, so it's more clear to the user?
`You are currently using Storybook XYZ but you have packages which are incompatible with it: -foo -bar`